From ccb009fc721e0b473bdf309cd3af3b51840bc55b Mon Sep 17 00:00:00 2001
From: Pascal Bach <pasci.bach@gmail.com>
Date: Thu, 6 Apr 2017 15:17:15 +0200
Subject: [PATCH] Excplicitly use utf-8 to read README file

We already know it is UTF-8 so we can make it explicit.
---
 setup.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/setup.py b/setup.py
index 57ced17..5126d24 100644
--- a/setup.py
+++ b/setup.py
@@ -7,7 +7,7 @@
 
 description = "An implementation of the WebSocket Protocol (RFC 6455)"
 
-with open(os.path.join(root_dir, 'README.rst')) as f:
+with open(os.path.join(root_dir, 'README.rst'), encoding="utf-8") as f:
     long_description = f.read()
 
 with open(os.path.join(root_dir, 'websockets', 'version.py')) as f:
